openmrs-19/src/main/java/org/motechproject/openmrs19/resource/impl/PersonResourceImpl.java

```diff
@@ -67,7 +67,7 @@ public void updatePersonName(Config config, String uuid, Person.Name name) {
     @Override
     public void updatePersonAddress(Config config, String uuid, Person.Address address) {
         String requestJson = buildGson().toJson(address);
-        postWithEmptyResponseBody(config, requestJson, "/person/{personUuid}/address/{addressUuid}", uuid, address.getUuid());
+        postWithEmptyResponseBody(config, requestJson, "/person/{personUuid}/address/{addressUuid}?v=full", uuid, address.getUuid());
```

Comment:
is this required ? This is POST request with empty response body. I suppose that this 'full' option refers to response body only.